src/OSPSuite.UI/UIConstants.cs

```diff
@@ -22,6 +22,7 @@ private static Color skinColorFor(string state)
       public const int TOOL_TIP_INITIAL_DELAY = 500;
       public const int TOOL_TIP_INITIAL_DELAY_LONG = 1500;
       public const string EMPTY_COLUMN = " ";
+      public const string DO_NOT_SHOW_THIS_AGAIN_UNTIL_RESTART = "Do not show this again until restart";
```

Comment:
texts are not stored in this file

Reply:
Let's put this into the developer documentation, so new devs know directly where to put string constants. What do you think?

Reply:
@peterdew place this in OSPSuite.Assets.Captions. You will be able to see the conventions there.